shared/types/api/pdao.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/rocket-pool/node-manager-core/eth"
 	"github.com/rocket-pool/rocketpool-go/v2/types"
+	sharedtypes "github.com/rocket-pool/smartnode/v2/shared/types"
```

Comment:
heh, i guess i forgot that `shared` still exists in v2... we should start trying to get rid of it, it's an antipattern.

Reply:
Sure. Let's do it on a separate PR?